src/basics.md

```diff
@@ -168,7 +158,7 @@ fn main() {
 }
 ```
 
-Alternatively, one can use the [`Range`](https://doc.rust-lang.org/rand/rand/distributions/range/struct.Range.html) distribution.
+Alternatively, one can use the [`Range`] distribution (otherwise known as uniform).
```

Comment:
I meant making "uniform" into a link pointing to something like:

https://en.m.wikipedia.org/wiki/Uniform_distribution_(continuous)